tests/utils.go

```diff
@@ -423,6 +440,169 @@ func CreateSecret(name string, data map[string]string) {
 	}
 }
 
+func GetListOfManifests(pathToManifestsDir string) []string {
```

Comment:
Maybe `filepath.Walk` is also helpful. Not sure it would really be much simpler, but maybe you want to have a look ...

Reply:
I changed that to `Walk`